pandas/tests/test_strings.py

```diff
@@ -1603,6 +1603,18 @@ def test_pad_fillchar(self):
                                    "fillchar must be a character, not int"):
             result = values.str.pad(5, fillchar=5)
 
+    def test_pad_width(self):
```

Comment:
you can just put them all in a single test method and iterate over the functions asserting that they raise